CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
 ## 2.2.2 (Unreleased)
+Adds support to the following HPE OneView resources:
+  - Added oneview_id_pool resource
```

Comment:
Since this release will add a new resource, it should likely be `2.3.0` instead of `2.2.2`